release/models/system/openconfig-system.yang

```diff
@@ -46,7 +46,13 @@ module openconfig-system {
     Section 4.c of the IETF Trust's Legal Provisions Relating
     to IETF Documents (http://trustee.ietf.org/license-info).";
 
-  oc-ext:openconfig-version "0.14.0";
+  oc-ext:openconfig-version "0.14.1";
+
+  revision "2022-07-25" {
+    description
+      "Add system software version.";
+    reference "0.14.1";
```

Comment:
```suggestion
    reference "0.15.0";
```

Since we are adding a leaf (vs. fixing a minor error/omission) we increment the minor version.

Reply:
I updated the reference version on this line, as well as the `oc-ext:openconfig-version` to both say `0.15.0`